index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portflio</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+    <div class="Mobile">
+        <ul class="header">
+            <li class="first"><a href="#">My Logo</a></li>
+            <li><a href="#">Portflio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
```

Comment:
Since you are still working on the mobile version, I think we need the hamburger icon instead of listing out the nav link as in the desktop version. 👍